frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/ssh/ssh-vm-details-page-testing.includes.ts

```diff
@@ -9,6 +11,9 @@ export default ({ vmName }) =>
         .then((link) => cy.visit(link.attr('href')));
       cy.byLegacyTestID(vmName).click();
       cy.byLegacyTestID('horizontal-link-Details').click();
+
+      // ensure vm is running before check ssh details
+      cy.get(detailsTab.vmStatus, { timeout: 600000 }).should('contain', 'Running');
```

Comment:
Can we add 'Cloning' to the VM_STATUS enum [1] and use it for the status here and in source-image.spec.ts? I'd eventually like to replace them anywhere else we may be hard-coding them, as well. I'd really like to avoid hard-coding constants as much as possible.

[1] https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests-cypress/const/index.ts#L44

Reply:
NP, I was thinking of using VM_STATUS while writing this.

Reply:
Thank you! =)